tests/test_relation.py

```diff
@@ -36,6 +36,12 @@ def setup_class(cls):
         cls.img = schema.Image()
         cls.trash = schema.UberTrash()
 
+    def test_class_help(self):
+        help(schema.TTest)
+
+    def test_instance_help(self):
```

Comment:
as before r.e. testing for definition in doc.

Reply:
done. Also moved into `test_declare`